examples/visualization/plot_publication_figure.py

```diff
@@ -44,7 +44,7 @@
 evoked.plot()
 
 stc.plot(views='lat', hemi='split', size=(800, 400), subject='sample',
-         subjects_dir=subjects_dir, initial_time=max_t)
+         subjects_dir=subjects_dir, initial_time=max_t, time_viewer=False)
```

Comment:
show_traces=False, too?

Reply:
normally, only `time_viewer=False` should be necessary but it's maybe clearer with `show_traces=False` too